airbyte-api/src/main/openapi/config.yaml

```diff
@@ -740,7 +740,7 @@ paths:
           content:
             application/json:
               schema:
-                $ref: "#/components/schemas/JobStatusRead"
+                $ref: "#/components/schemas/JobInfoRead"
```

Comment:
@Jamakase , note this change. it used to be pretty much just be a boolean (though it was technically an enum, the enum just had two value). this version there are more values in the enum, but i think for the way it is used in the ui, you can pretty much just do `status === JobStatus.SUCCEEDED` to determine if the job was a success or not, so hopefully it is not really more complicated, even though there is more in this struct than there was before.